pkg/components/component.go

```diff
@@ -60,14 +61,18 @@ type baseComponent struct {
 	labeller *labeller.Labeller
 }
 
-// GetName returns component's name, which is used as an identifier in component management
+// GetFullName returns component's name, which is used as an identifier in component management
 // and for mentioning in logs.
 // For example for master component name is "Master",
 // For data node name looks like "DataNode<NameFromSpec>".
-func (c *baseComponent) GetName() string {
+func (c *baseComponent) GetFullName() string {
 	return c.labeller.GetFullComponentName()
 }
 
+func (c *baseComponent) GetShortName() string {
+	return c.labeller.GetInstanceGroup()
+}
```

Comment:
Are you sure that `default` won't be returned here if name not set? Maybe it should be take the code from `getGroupName()` this function to be reused there?